dockers/docker-orchagent/swssconfig.sh

```diff
@@ -55,7 +55,12 @@ HWSKU=${HWSKU:-`sonic-cfggen -d -v "DEVICE_METADATA['localhost']['hwsku']"`}
 SYSTEM_WARM_START=`sonic-db-cli STATE_DB hget "WARM_RESTART_ENABLE_TABLE|system" enable`
 SWSS_WARM_START=`sonic-db-cli STATE_DB hget "WARM_RESTART_ENABLE_TABLE|swss" enable`
 if [[ "$SYSTEM_WARM_START" == "true" ]] || [[ "$SWSS_WARM_START" == "true" ]]; then
-  exit 0
+    # Always load ipinip.json even on warm boot
```

Comment:
This will apply ipinip config regardless of what in IPiniP is being changed. 

Do you think any and all changes in ipinip will be non disruptive to the tunnel? In other words, will all changes (not just limited to ECN mode, DSCP mode) lead to SET only operations, and no DEL?